content/articles/searching-and-sorting-techniques/index.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /searching-and-sorting-techniques /
+title: Searching and sorting techniques 
+description: In this article, we will learn about various sorting and searching techniques and their implementation in data structures and algorithms.
+author: jairus-onkundi
+date: 2021-09-21T00:00:00-13:35
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+    - url: /engineering-education/searching-and-sorting-techniques/hero.jpg
+      alt: Searching and sorting techniques 
+---
+
+### Introduction
+
+Searching is the process of determining the position of an item in a list. This is a crucial aspect of many data structure techniques, because we can only execute one action on an item if and only if we locate the item. To determine if an element is present in a collection of components, many methods have been devised as we will discuss them in this article.

```

Comment:
```suggestion
Searching is determining the position of an item in a list. This is a crucial aspect of many data structure techniques, because we can only execute one action on an item if and only if we locate the item. To determine if an element is present in a collection of components, many methods have been devised, as we will discuss them in this article.

```